clients/python/pyproject.toml

```diff
@@ -3,16 +3,16 @@ name = "chromadb-client"
 dynamic = ["version"]
 
 authors = [
-  { name="Jeff Huber", email="jeff@trychroma.com" },
-  { name="Anton Troynikov", email="anton@trychroma.com" }
+  { name = "Jeff Huber", email = "jeff@trychroma.com" },
```

Comment:
whats with the formatting thrashing here lol - i've seen 2-3 prs where this file is only getting formatted.

Reply:
adding httpx here ;)

I just tested and VS Code added the formatting changes upon save, I assume because `.vscode/settings.json->editor.formatOnSave` is true
not sure what formatter it's actually using for that though